shell/platform/darwin/ios/framework/Headers/FlutterViewController.h

```diff
@@ -24,6 +24,13 @@ FLUTTER_EXPORT
 
 - (void)handleStatusBarTouches:(UIEvent*)event;
 
+/**
+ Registers a callback that will be invoked when the Flutter view has been rendered.
+
+ Replaces an existing callback. Use a `nil` callback to unregister the existing one.
```

Comment:
Add more details. Would it be called again if you set, unset and set again on the same VC? (seems like no). Would it be called if you set it too late? (seems like no).